quickstart/test/compose-test.zsh

```diff
@@ -39,9 +39,9 @@ TESTDIR="$(mktemp -d -t "${BASENAME%.*}.${DATESTAMP}.XXX")"
 if [[ -z "${ZITI_QUICK_DIR:-}" ]]; then
     ZITI_QUICK_DIR="$(realpath "${DIRNAME}/..")"
 fi
-# if unset, set ZITI_QUICK_IMAGE_TAG to this run's dirname
-if [[ -z "${ZITI_QUICK_IMAGE_TAG:-}" ]]; then
-    ZITI_QUICK_IMAGE_TAG=$(basename "${TESTDIR}")
+# if unset, set ZITI_QUICK_TAG to this run's dirname
```

Comment:
adopt the name used by the all-in-one quickstart for consistency

Reply:
This change was moved to [another branch](https://github.com/openziti/ziti/pull/1749) when this branch's changes were split to make them easier to review.